Objects/EventItem/EventItem.gd

```diff
@@ -0,0 +1,28 @@
+extends Node2D
+
+export(int) var gold_reward = 65
+export(float) var move_speed = 100
+export(Vector2) var start_position = Vector2()
+
+
+
+func _ready():
+	var value = randi()%800+160
+	set_start_position(value,550)
+
+func _process(delta):
+	move(delta)
+	
+func move(delta):
+	position.x -= move_speed * delta
```

Comment:
For me it's a little unintuitive, when we subtracting speed from position. I saw, that in DwarfInBalloon.gd line 7 we are moving with negative delta. That's unintuitive too. I recomend to write that line as below:
```gdscript
position.x += move_speed * delta
```
and set move_speed to negative value (in example `Vector2(-200, 0)`)

It will be easier to read and write that code.